packages/remark-mdx/test/index.js

```diff
@@ -1302,6 +1302,16 @@ test('stringify', function (t) {
     'should serialize an expression attribute (element)'
   )
 
+  t.equal(
+    basic.stringify(
+      u('paragraph', [
+        u('link', {url: 'https://mdxjs.com'}, [u('text', 'https://mdxjs.com')])
+      ])
+    ),
+    'https://mdxjs.com',
```

Comment:
This will only work in GFM though, unfortunately.

Maybe copying the [default visitor](https://github.com/remarkjs/remark/blob/19a27c30b13baa3a9e864e528c6a0b901a5fc918/packages/remark-stringify/lib/visitors/link.js#L46-L49) w/o L46-L49 instead is better for us here?

Reply:
That works for me if you think that's best 👍

Reply:
Done!